src/dev/build/tasks/os_packages/service_templates/sysv/etc/init.d/kibana

```diff
@@ -151,9 +155,15 @@ case "$1" in
     fi
     exit $code
     ;;
+  condrestart|try-restart)
```

Comment:
why the addition of two options `condrestart` and `try-restart` ? Isn't `try-restart` enough?

Reply:
This was copy-paste from someone else's script. There are essentially two different verbs in common use for this action. Since I'm adding the thing, it seemed reasonable to support both for humans as opposed to just for this thing itself.

I don't have a strong opinion about this. My systems going forward are Debian derivatives with systemd...

Reply:
https://www.cyberciti.biz/faq/linux-unix-sysvinit-services-restart-vs-reload-vs-condrestart/ for lack of a better reference

Reply:
@jsoref I also found docs about this [here](https://fedoraproject.org/wiki/EPEL:SysVInitScripts?rd=Packaging:SysVInitScript#condrestart_and_try-restart). I'm fine with it!